### PR TITLE
Add missing SUMMARY

### PR DESCRIPTION
**Pull request by snetting**
On running haikuporter, whilst populating the repository a number of recipes report errors due to missing SUMMARY definitions.  The Wiki states: "The old system did not use SUMMARY. All recipes need a SUMMARY of 70 characters or less".  I notice these packages are also missing from the HaikuDepot GUI tool - I assume due to this issue.

This pull request adds a SUMMARY to the problem recipes:-

games-rpg/freedroid/freedroid-1.0.2.recipe
sys-apps/hgrep
net-libs/libyahoo2
dev-db/mysql
dev-python/pychart, pydispatcher, pyopengl, pyxml, rdflib

### CODE REVIEW

File: dev-db/mysql/mysql-5.0.83.recipe [hunk 1]
```diff
@@ -1,10 +1,12 @@
-DESCRIPTION="SQL database server."
+SUMMARY="SQL Database server"
+DESCRIPTION="A very fast and reliable SQL database server"
```
Comment (fbrosson) [inline dev-db/mysql/mysql-5.0.83.recipe]: Maybe:
```diff
-SUMMARY="SQL Database server"
+SUMMARY="A SQL Database server"
-DESCRIPTION="A very fast and reliable SQL database server"
+DESCRIPTION="A very fast and reliable SQL database server."
```
Yep, that's a bit special. We only put full stops in `DESCRIPTION`.

File: dev-db/mysql/mysql-5.0.83.recipe [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 CHECKSUM_MD5="051392064a1e32cca5c23a593908b10e"
 REVISION="1"
 STATUS_HAIKU="untested"
 DEPEND="sys-libs/readline >= 5.2"
 
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+        mysql$secondaryArchSuffix = $portVersion"
```
Comment (fbrosson) [inline dev-db/mysql/mysql-5.0.83.recipe]: This recipe is still using the old format has the so maybe we should remove it from this PR and create a separate PR for it.
(The `STATUS_HAIKU` variable comes from the old format.)

BTW, that's a very good idea to have one commit per port. We'll be able to merge this PR with the "Rebase and merge" method.

File: dev-python/pychart/pychart-1.39.recipe [hunk 3]
```diff
@@ -1,4 +1,5 @@
-DESCRIPTION="PyChart is a Python library for creating high quality charts."
+SUMMARY="Library for Python"
+DESCRIPTION="PyChart is a Python library for creating high quality charts"
```
Comment (fbrosson) [inline dev-python/pychart/pychart-1.39.recipe]: Maybe:
```diff
-SUMMARY="Library for Python."
+SUMMARY="A library for creating charts with Python"
-DESCRIPTION="PyChart is a Python library for creating high quality charts"
+DESCRIPTION="PyChart is a Python library for creating high quality charts."
```

File: dev-python/pychart/pychart-1.39_bzr.recipe [hunk 4]
```diff
@@ -1,4 +1,5 @@
-DESCRIPTION="PyChart is a Python library for creating high quality charts."
+SUMMARY="Library for Python"
+DESCRIPTION="PyChart is a Python library for creating high quality charts"
```
Comment (fbrosson) [inline dev-python/pychart/pychart-1.39_bzr.recipe]: Maybe:
```diff
-SUMMARY="Library for Python."
+SUMMARY="A library for creating charts with Python"
-DESCRIPTION="PyChart is a Python library for creating high quality charts"
+DESCRIPTION="PyChart is a Python library for creating high quality charts."
```

**Comment by snetting** [comment]
Closing as many of these packages are broken and require additional effort to fix.  I'll work on this package by package.

**Comment by scottmc** [comment]
Also you can run Haikuports --lint to see a list of all recipes with various issues such as No Summary.  Feel free to fix those up one by one.  Most of them worked at some point  in the past.

**Pull request closed without merging**